EIPS/eip-3675.md

```diff
@@ -88,6 +88,7 @@ Beginning with `TRANSITION_BLOCK`, the block validity conditions **MUST** be alt
 * Remove all validation rules that are evaluated over the list of ommers and each member of this list.
 * Add verification of the fields noted in [block structure](#block-structure) as their specified constant value.
 * Add verification of block validity with respect to the PoS consensus rules, i.e. assert that there **MUST** be a corresponding `POS_CONSENSUS_VALIDATED` event for the block to consider it valid.
+* Add verification of the block's **`extraData`** field. i.e. the length of the block's **`extraData`** **MUST** be less or equal to 32 bytes.
```

Comment:
Suggest "32 bytes" be defined as a constant earlier in the spec. e.g. `MAX_BYTES_PER_EXTRA_DATA`